lms/static/coffee/src/instructor_dashboard/membership.coffee

```diff
@@ -670,6 +670,564 @@ class AuthList
       success: (data) -> cb?(data)
       error: std_ajax_err => @$request_response_error.text gettext "Error changing user's permissions."
 
+class EmailSelectors
+  constructor: (@$container, @$section, params={}) ->
```

Comment:
I am using it later on in the EmailWidget class:

 for email_list in @email_lists
      email_list.$container.addClass 'active'

However I have been mixing @$ and this. so I changed all of them to standardize on @$